metronome/hotstuff/service/src/io/iohk/metronome/hotstuff/service/tracing/SyncTracers.scala

```diff
@@ -0,0 +1,22 @@
+package io.iohk.metronome.hotstuff.service.tracing
+
+import cats.implicits._
+import io.iohk.metronome.tracer.Tracer
+import io.iohk.metronome.hotstuff.consensus.basic.Agreement
+
+case class SyncTracers[F[_], A <: Agreement](
+    queueFull: Tracer[F, A#PKey],
+    error: Tracer[F, Throwable]
```

Comment:
WDYT about tracing each network message sent/received?

Reply:
I thought that would be good too, then I remembered this is [already happening](https://github.com/input-output-hk/metronome/pull/13/files#diff-dec301a3a2b7454706e960cea945216df1c5c0376b6327b9e47009fb842540b8R46-R52) on the network layer: https://github.com/input-output-hk/metronome/pull/13

The only difference is that it would be of the higher level message kind (what is called `DuplexMessage` now; I wish we had a better name for it). It should be easy to patter match on it to, say, log just the sync messages.